draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -128,7 +125,7 @@
     <section title='Terminology' anchor='Terminology'>
       <t>
 	This specification uses terms defined in
-	the CBOR Web Token <xref target="CWT"/>,
+	the CBOR Web Token <xref target="RFC8392"/>,
```

Comment:
Please update this from `CBOR Web Token <xref target="RFC8392"/>` to `CBOR Web Token (CWT) <xref target="RFC8392"/>`.